src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
+    @Max(value = 256)
```

Comment:
I would add Min annotation as well

Reply:
In my opinion, It's redundant. Because movie titles can be from one or more characters, but they can't be NULL